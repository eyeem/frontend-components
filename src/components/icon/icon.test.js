import React from 'react';

import { render } from '@testing-library/react';

import Icon from './index';

test('it renders default icon', () => {
  const { container } = render(<Icon type="like" />);
  const component = container.firstChild;
  expect(component).toBeTruthy();
  expect(component.nodeName).toBe('svg');
  expect(component.firstChild.nodeName).toBe('path');
  expect(component).toMatchSnapshot();
});
