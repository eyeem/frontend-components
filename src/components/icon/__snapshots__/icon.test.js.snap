// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`it renders default icon 1`] = `
<svg
  height="16"
  viewBox="0 0 24 24"
  width="16"
>
  <path
    d="M2.556 10.31C3.081 15.03 12 20.802 12 20.802s8.919-5.246 9.444-10.492c.524-5.247-2.694-7.437-6.296-6.296C13.597 4.504 12 6.742 12 6.742s-1.597-2.237-3.148-2.728c-3.602-1.14-6.82 1.574-6.296 6.296z"
    fill="none"
    stroke="#fff"
    stroke-width="2"
  />
</svg>
`;
